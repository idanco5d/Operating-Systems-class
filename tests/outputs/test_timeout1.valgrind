==2930== Memcheck, a memory error detector
==2930== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==2930== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==2930== Command: /mnt/c/Users/illus/os/hw/1/smash
==2930== Parent PID: 2924
==2930== 
==2930== 
==2930== FILE DESCRIPTORS: 4 open (3 std) at exit.
==2930== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_timeout1.valgrind
==2930==    <inherited from parent>
==2930== 
==2930== 
==2930== HEAP SUMMARY:
==2930==     in use at exit: 0 bytes in 0 blocks
==2930==   total heap usage: 69 allocs, 69 frees, 81,646 bytes allocated
==2930== 
==2930== All heap blocks were freed -- no leaks are possible
==2930== 
==2930== For lists of detected and suppressed errors, rerun with: -s
==2930== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
