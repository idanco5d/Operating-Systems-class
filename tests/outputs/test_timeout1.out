smash> smash> smash> [1] timeout 2 sleep 10& : 9990 0 secs
smash> smash> smash: sending SIGKILL signal to 0 jobs:
